pulpcore/tests/functional/api/using_plugin/constants.py

```diff
@@ -9,7 +9,7 @@
     CONTENT_PATH
 )
 
-FILE_CONTENT_PATH = urljoin(CONTENT_PATH, 'file/files/')
+FILE_CONTENT_PATH = urljoin(CONTENT_PATH, 'pulp_file/file/files/')
```

Comment:
This is a wrong path but changes in the plugin are needed. I changed it to show that tests are passing.